shell/platform/embedder/embedder.cc

```diff
@@ -125,6 +125,14 @@ FlutterResult FlutterEngineRun(size_t version,
     };
   }
 
+  std::function<bool()> make_resource_current_callback = nullptr;
+  if (config->open_gl.make_resource_current != nullptr) {
```

Comment:
This needs to be wrapped with the SAFE_ACCESS macro.

Reply:
Done